test/Sentry.Tests/SentryOptionsTests.verify.cs

```diff
@@ -3,9 +3,12 @@ namespace Sentry.Tests;
 [UsesVerify]
 public partial class SentryOptionsTests
 {
-    [Fact]
+    [SkippableFact]
     public Task Integrations_default_ones_are_properly_registered()
     {
+        // Windows additionally adds `WinUIUnhandledExceptionIntegration`
+        Skip.If(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
```

Comment:
should we have two tests then? one that tests we have this on Windows? Instead of skipping

Reply:
I had that but it messed with the verify. I guess I need to run this on windows then too to generate the expected file?

Reply:
yeah you need to have a file per platform, just saw this on another PR:
![image](https://github.com/getsentry/sentry-dotnet/assets/1633368/e1767332-8024-476c-818d-d5e229d01271)

Reply:
one of those per OS I imagine